modules/oe_theme_content_event/src/Plugin/ExtraField/Display/InfoDisclosureExtraFieldBase.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_theme_content_event\Plugin\ExtraField\Display;
+
+use Drupal\Component\Datetime\TimeInterface;
+use Drupal\Core\Datetime\DateFormatterInterface;
+use Drupal\oe_time_caching\Cache\TimeBasedCacheTagGeneratorInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Base class for fields that require display in exact time.
+ */
+abstract class InfoDisclosureExtraFieldBase extends DateAwareExtraFieldBase {
```

Comment:
I think this class name doesn't reflect exactly what it is doing like with dates or livestream depending on the functions it has.

Reply:
Currently, this class is used for Livestream-related fields but idea is to make the approach more generic for reusing in registration buttons fields (which also should be displayed in exact time).
I would stay for now with this naming.